src/components/AddLayerModal/index.tsx

```diff
@@ -81,6 +84,28 @@ export const AddLayerModal: React.FC<AddLayerModalProps> = ({
     }
   };
 
+  /**
+   * Checks and completes a WMS GetCapabilities URL.
+   * The params request, service and version will be completed.
+   * @param url The input url
+   * @returns The completed URL
+   */
+  const completeUrl = (url: string) => {
```

Comment:
We might want to reuse [this method](https://github.com/terrestris/base-util/blob/master/src/UrlUtil/UrlUtil.ts#L129) from the base-util instead.